types/newman/index.d.ts

```diff
@@ -20,6 +20,8 @@ export interface NewmanRunOptions {
     /** An environment JSON / file path for the current collection run. */
     environment?: VariableScope | VariableScopeDefinition | string;
     /** A globals JSON / file path for the current collection run. */
+    envVar?: Collection | CollectionDefinition | string;
+    /** An override to environment variables.  See: https://github.com/postmanlabs/newman/blob/develop/lib/run/options.js */
```

Comment:
Was this comment supposed to go above the new property?

Reply:
ooops,  my bad.  fixed